test/hapi_server.js

```diff
@@ -56,11 +56,20 @@ describe('watch Hapi server', function () {
     });
   });
 
+  after(function (done) {
+    server.stop(done);
+  });
+
   it('should log response time', function (done) {
     eventLogger.logger.info = function(log_event) {
+      if (log_event.log_type === 'request') {
```

Comment:
why the changes on these tests?

Reply:
Not sure why it works before! in the `request` is when the clock starts.

Reply:
Might have had a 0 as default and now it has changed? Do it still works if you remove the change?

Reply:
it's tested against each specific version of hapi!